### PR TITLE
Internal: fix for node 14

### PR DESCRIPTION
**Pull request by SethRogers7420**
This PR fixes schemats not working on Node 14 by bumping some library versions, primarily pg-promise.

I added instructions on testing this at the bottom of the Readme.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -108,5 +108,5 @@ export async function typescriptOfSchema (db: Database|string,
     return processedResult.dest
 }
 
-export {Database, getDatabase} from './schema'
-export {Options, OptionValues}
+export { Database, getDatabase } from './schema'
+export { Options, OptionValues }
```
Comment (SethRogers7420) [inline src/index.ts:112]: Linter was mad at me.

Note that we also have some test failures from manual changes to this code that were never fixed.

File: package.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "@types/mocha": "^2.2.39",
     "@types/mysql": "0.0.33",
     "@types/mz": "^0.0.31",
-    "@types/node": "^6.0.77",
+    "@types/node": "^16.9.1",
```
Comment (tallpaul-embark) [inline package.json]: should this be set to the same version of node we are on?

Comment (SethRogers7420) [reply package.json]: Oops yes, I forgot we aren't on the latest.
Thanks for the catch!

Comment (SethRogers7420) [reply package.json]: Fixed

**Review by ghost — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "schemats",
-  "version": "3.0.7-embark",
```
Comment (parsingphase) [inline package.json:3]: Being semantically pedantic this should be `3.0.7-embark-2` unless we've updated against the source library's `3.0.8`

Comment (SethRogers7420) [reply package.json:3]: The original schemats is 3.0.3, this seems to be what has been done to bump the version before 🤷

Comment (parsingphase) [reply package.json:3]: Looks like the original (https://github.com/SweetIQ/schemats) is abandonware, so TBH we should probably start numbering it ourselves.

Comment (ghost) [reply package.json:3]: In the past we considered taking over this repo and publishing updates to it so we tried to keep the version in line with what would be used in the library. We never did this but have kept the same versioning. The `-embark` now just denotes it is our version.

File: package.json [hunk 4]
```diff
@@ -69,8 +69,8 @@
     "lodash": "^4.17.4",
     "mysql": "^2.13.0",
     "mz": "^2.6.0",
-    "pg-promise": "^6.3.6",
-    "typescript": "^2.7.1",
+    "pg-promise": "^10.11.0",
+    "typescript": "^4.4.3",
```
Comment (parsingphase) [inline package.json]: projectaussie has:

     "typescript": "4.2.3",
     "pg-promise": "10.10.2",
     
We should likely not require higher than those.

**Review by parsingphase — CHANGES_REQUESTED** [review]
I think we need to further soften / lower version reqs as commented

File: package.json (hunk 4 above)
Comment (SethRogers7420) [reply package.json]: Done!

**Review by parsingphase — APPROVED** [review]

**Pull request merged**